.github/workflows/codeql-analysis.yml

```diff
@@ -24,36 +24,37 @@ on:
   schedule:
     - cron: '45 10 * * 1'
 
-permissions:
```

Comment:
can we set here

```
permissions: {}
```


then we drop all for the workflow and just set for the job?